loginsightwebhookdemo/vrealizeorchestrator.py

```diff
@@ -7,17 +7,19 @@
 
 __author__ = "John Dias"
 __license__ = "Apache v2"
-__version__ = "1.1"
+__version__ = "1.2"
 
 
-# vRealize Orchestrator server workflow hostname (or hostname:port)
+# vRealize Orchestrator server workflow hostname:port (default port is 8281)
 VROHOSTNAME = ''
-
+# For some labs, using self-signed will result in error during request due to cert check
+# flip this flag to False to bypass certificate checking in those cases
+VERIFY = True
 
 
 @app.route("/endpoint/vro/<WORKFLOWID>", methods=['POST'])
-@app.route("/endpoint/vro/<WORKFLOWID>/<ALERTID>", methods=['PUT'])
```

Comment:
This was added to support vROps REST plugin notifications. Arguably, vROps can send directly to vRO so leveraging the shim is not necessary, but since this was added to all other shims, I though it was a good idea. Thoughts?